dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -143,6 +143,10 @@ separately this command:
 breeze release-management prepare-provider-documentation qubole
 ```
 
+In case you want to **just** regenerate the documentation because you fixed something in the templates, add
+`--reapply-templates` flag to the command above. If you want to just update min airflow version for all
+packages, you can use `--only-min-version-update` flag - this will only update the min version in the
+`__init__.py` files of the packages and package documentation without bumping the provider versions.
```

Comment:
If I understand this right...
`breeze release-management prepare-provider-documentation --only-min-version-update`
should be done as a step that is not really part of the release process? We normally update the min airflow version on the promised date regardless if we cut release afterwards

Reply:
Correct.

Reply:
Cool 
I didnt review all the code but from doc point of view it miss some stuff. It should have a parameter of what version to uodate to.

We have cases where providers have different min version so this cant be just bumping automaticly. It needs to know which providers wo skip

Reply:
Updated the docs.  Also clarified that you do NOT have to skip any providers when you use `-only-min-version-update`. It works in a smart way - it will only increase the min_airflow_version if it is not set to a higher one.

For example when you have common.io provider with min_airflow_version = 2.8, it will stey at 2.8 when we bump min version to 2.6. It will also stay at 2.8 when we bump it to 2.7, and then when we bump everything to 2.8, it will stay at 2.8, and when we bump to 2.9 - it will get bumped to 2.9 lile everything else, wich is precisely what we want.